quick_start/kusto_sample_app.py

```diff
@@ -0,0 +1,334 @@
+        print("Failed to alter the ingestion policy!")
+        print("This could be the result of insufficient permissions.")
+        print("The sample will still run, though ingestion will be delayed for 5 minutes")
+
+    # Todo - Learn More:
+    #  Kusto batches data for ingestion efficiency. The default batching policy ingests data when one of the following conditions are met:
+    #   1) more then a 1000 files were queued for ingestion for the same table by the same user
+    #   2) more then 1GB of data was queued for ingestion for the same table by the same user
+    #   3) More then 5 minutes have passed since the first file was queued for ingestion for the same table by the same user
+    #  In order to speed up this sample app, we attempt to modify the default ingestion policy to ingest data after 10 seconds have passed
+    #  For additional information on customizing the ingestion batching policy see:
+    #   https://docs.microsoft.com/en-us/azure/data-explorer/kusto/management/batchingpolicy
+    #  You may also skip the batching for some files using the FlushImmediatly property, though this option should be used with care as it is inefficient
+    wait_for_user()
+
+    # Todo - Learn More: For additional information on Kusto Query Language see: https://docs.microsoft.com/en-us/azure/data-explorer/write-queries
+    print("")
+    print(f"Initial row count for '{databaseName}.{tableName}' is:")
+    run_query(kusto_client, databaseName, f"{tableName} | summarize count()")
+    wait_for_user()
+
+    # Todo - Learn More: For additional information on how to ingest data to Kusto in Python see:
+    #  https://docs.microsoft.com/en-us/azure/data-explorer/python-ingest-data
+    if csvSample is not None:
+        print("")
+        print(f"Attempting to ingest '{csvSample}'")
+        ingest_data_from_file(ingest_client, databaseName, tableName, csvSample, DataFormat.CSV)
+        wait_for_user()
+
+    if jsonSample is not None:
+        print("")
+        print(f"Attempting to create a json mapping reference named '{jsonMappingRef}'")
+        # Todo - Tip: this is commonly a one-time command
+        mapping_command = f".create-or-alter table {tableName} ingestion json mapping '{jsonMappingRef}' '{jsonMapping}'"
+        mapping_exists = run_control_command(kusto_client, databaseName, mapping_command)
+        if not mapping_exists:
+            print(f"failed to create a json  mapping reference named {jsonMappingRef}")
+            print(f"skipping json ingestion")
+
+        # Todo - learn more:  for more information about providing inline mappings or mapping references see:
+        #  https://docs.microsoft.com/en-us/azure/data-explorer/kusto/management/mappings
+
+        wait_for_user()
+
+        if mapping_exists:
+            print("")
+            print(f"Attempting to ingest '{jsonSample}'")
+            ingest_data_from_file(ingest_client, databaseName, tableName, jsonSample, DataFormat.MULTIJSON, jsonMappingRef)

```

Comment:
add some comment to suggest to always use MULTIJSON when ingesting Json

Reply:
Good?
```
# Tip: When ingesting json files, if a each row is represented by a single line json, use MULTIJSON format even if the file only includes one line.
# When the json contains whitespace formatting, use SINGLEJSON. IN this case only one row/json per file is allowed.
```